# SIB course 2024 - Biodiversity bioinformatics

_From large-scale phylogenomics to gene families and functions_

This repository contains the materials for the practical exercises of Day 2 of the course: **Orthology-focused arthropod comparative genomics**.

It aims to:
- Explain the principles of graph-based orthology delineation using OrthoDB as an example
- Show how to browse and programmatically query OrthoDB
- Show how to use BUSCO to assess genomics data quality
- Show how to formulate comparative genomics questions, develop and apply approaches to address them (with a focus on using orthology data), and then critically interpret them

This course is taught by Robert Waterhouse (SIB).


## Lectures

All the lecture materials are provided here in this [PDF](https://github.com/rmwaterhouse/biodivinfo/blob/main/SIB%20Biodiversity%20Bioinformatics%20DAY2.pdf).

## Practicals

Once you have created a GitHub account and linked it to GitPod, you can open this repository with GitPod and start the practicals. You will find help on how to do so [here](https://www.gitpod.io/docs/introduction/getting-started#step-1-your-first-workspace).


## Software and Resources

These links are provided for information only, they are not needed now for the practical but if you finish any exercises early you may want to investigate the tools/resources a little further by checking out their websites, user guides, and/or their main publications.

- OrthoDB - orthology database --> 
[Website](https://www.orthodb.org/); [Publication](https://academic.oup.com/nar/article/49/D1/D389/5983625); [Userguide](https://www.ezlab.org/orthodb_userguide.html)
- BUSCO - Benchmarking Universal Single-Copy Orthologues --> 
[Website](https://busco.ezlab.org/); [Publication](https://academic.oup.com/mbe/article/38/10/4647/6329644); [Userguide](https://busco.ezlab.org/busco_userguide.html)
- MAFFT - multiple sequence alignment tool --> 
[Website](https://mafft.cbrc.jp/alignment/software/); [Publication](https://academic.oup.com/mbe/article/30/4/772/1073398); [Userguide](https://mafft.cbrc.jp/alignment/software/manual/manual.html)
- TrimAl - alignment filtering tool --> 
[Website](https://vicfero.github.io/trimal/); [Publication](https://academic.oup.com/bioinformatics/article/25/15/1972/213148); [Userguide](https://vicfero.github.io/trimal/whatcanido.html)
- RAxML - maximum likelihood phylogenies --> 
[Website](https://cme.h-its.org/exelixis/web/software/raxml/); [Publication](https://academic.oup.com/bioinformatics/article/22/21/2688/251208); [Userguide](https://cme.h-its.org/exelixis/resource/download/NewManual.pdf)
- Treerecs - tree reconciliation software --> 
[Website](https://project.inria.fr/treerecs/); [Publication](https://project.inria.fr/treerecs/publications/); [Userguide](https://project.inria.fr/treerecs/treerecs-options/)
- CAFE - computational analysis of gene family evolution --> 
[Website](https://github.com/hahnlab/CAFE5); [Publication](https://academic.oup.com/bioinformatics/article/36/22-23/5516/6039105); [Userguide](https://github.com/hahnlab/CAFE5)
- iTOL - online phylogenetic tree viewer --> 
[Website](https://itol.embl.de/); [Publication](https://academic.oup.com/nar/article/49/W1/W293/6246398); [Userguide](https://itol.embl.de/help.cgi)
- Chronos function of the ape package in R --> 
[Website/Userguide](https://rdrr.io/cran/ape/man/chronos.html); [Publication](https://www.sciencedirect.com/science/article/pii/S1055790313000651)
- JalView: for multiple sequence alignment editing, visualisation and analysis --> 
[Website](https://www.jalview.org/); [Publication](https://academic.oup.com/bioinformatics/article/25/9/1189/203460)
